singolo/index.html

```diff
@@ -0,0 +1,221 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>SINGOLO</title>
+  </head>
+  <body>
+    <header class="header" id="header">
+      <div class="header-content">
+        <div class="header__logo">
+          <div class="burger-menu">
+            <div class="line"></div>
+            <div class="line"></div>
+            <div class="line"></div>
+          </div>
+          <h1>singolo<span>*</span></h1>
```

Comment:
not necessary to have h1 tag in this case

Reply:
fixed